### PR TITLE
Use Mozilla's own It's a Trap test page

### PR DESCRIPTION
**Pull request by stephendonner**

### CODE REVIEW

**Comment by davehunt** [comment]
Looks good. Could you look into the ESR failure?

**Comment by stephendonner** [comment]
@davehunt yeah, I'd be happy to

File: tests/test_browser_model.py [hunk 1]
```diff
@@ -50,6 +50,6 @@ def test_tracking_protection_shield(foxpuppet, selenium):
     """Tests if the tracking protection icon displays."""
     browser = foxpuppet.browser
     assert not browser.navbar.is_tracking_shield_displayed
-    selenium.get('https://www.washingtonpost.com/')
-    WebDriverWait(selenium, timeout=5).until(
+    selenium.get('https://itisatrap.org/firefox/its-a-tracker.html')
+    WebDriverWait(selenium, timeout=8).until(
```
Comment (davehunt) [inline tests/test_browser_model.py]: (in response to commit message) this is seconds

**Comment by davehunt** [comment]
Taking care of this in #195, where I've cherry-picked your commit @stephendonner.

**Pull request closed without merging**